.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a bug report
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Summary**
+A clear and concise description of what the bug is.
+
+**Steps to reproduce**
+Steps someone else can follow to reproduce the behavior.
+1. ...
+2. ...
+3. ...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Actual behavior**
+A clear and concise description of what happened instead.
+
+**Attachments**
+If applicable, add screenshots or videos to help explain your problem.
+
+**Environment**
+ - OS:
+ - Browser:
+ - Version:
```

Comment:
I would make this specify `xmtp-js` version, just to avoid confusion. Maybe they have their own app versions, or decide to point to the version of the chat app.